homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -337,11 +469,131 @@ class PowerViewShade(PowerViewShadeBase):
         | CoverEntityFeature.SET_POSITION
     )
 
+
+class PowerViewShadeTDBU(PowerViewShade):
+    """Representation of a PowerView shade with top/down bottom/up capabilities."""
+
+    @property
+    def current_cover_position_primary(self):
+        """Return the current position of cover."""
+        return hd_position_to_hass(self.position_primary)
+
+    @property
+    def current_cover_position_secondary(self):
+        """Return the current position of cover."""
+        return hd_position_to_hass(self.position_secondary)
+
+    @property
+    def get_transition_steps(self):
+        """Return the steps to make a move."""
+        return (
+            self.current_cover_position_primary + self.current_cover_position_secondary
+        )
+
+    async def _async_move(self, target_hass_position):
+        """Move the shade to a position."""
+        # custom move command to prevent excessive refresh on tdbu
+        self._async_cancel_scheduled_transition_update()
+        self._forced_resync = async_call_later(
+            self.hass, RESYNC_DELAY, self._async_force_resync
+        )
+
+        await self._shade.move(self._set_shade_postion(target_hass_position))
+
+        self._async_cover_transition_complete(
+            self.current_cover_position, target_hass_position
+        )
```

Comment:
## Base class
```python3
    async def _async_move(self, target_hass_position):
        """Move the shade to a position."""

        self._async_cover_transition_begin(
            self.current_cover_position, target_hass_position
        )

        self._async_update_from_command(
            await self._shade.move(self._set_shade_postion(target_hass_position))
        )

        self._async_cover_transition_complete(
            self.current_cover_position, target_hass_position
        )

```

## PowerViewShadeTDBU
```python3
    async def _async_move(self, target_hass_position):
        """Move the shade to a position."""
        # custom move command to prevent excessive refresh on tdbu
        self._async_cancel_scheduled_transition_update()
        self._forced_resync = async_call_later(
            self.hass, RESYNC_DELAY, self._async_force_resync
        )

        await self._shade.move(self._set_shade_postion(target_hass_position))

        self._async_cover_transition_complete(
            self.current_cover_position, target_hass_position
        )
```

It look like you could override `_async_cover_transition_begin` in
`PowerViewShadeTDBU` instead of making a new `_async_move` unless
you are not consuming the return value of `self._shade.move`
into `_async_update_from_command` because its not usable for
`PowerViewShadeTDBU`?